vms/platformvm/service.go

```diff
@@ -510,6 +510,80 @@ func (s *Service) GetUTXOs(_ *http.Request, args *api.GetUTXOsArgs, response *ap
 	return nil
 }
 
+/*
+ *****************************************************
+ ******************* Get Subnet **********************
+ *****************************************************
+ */
+
```

Comment:
I don't feel like these comments add anything anymore... (and they are really only around some of the old functions)

```suggestion
```